packages/backend/src/routers/projectRouter.ts

```diff
@@ -377,3 +377,76 @@ projectRouter.get(
         }
     },
 );
+
+projectRouter.get('/projectAccess', isAuthenticated, async (req, res, next) => {
+    try {
+        const results = await projectService.getProjectAccess(
+            req.user!,
+            req.params.projectUuid,
+        );
+        res.json({
+            status: 'ok',
+            results,
+        });
+    } catch (e) {
+        next(e);
+    }
+});
+
+projectRouter.post(
+    '/projectAccess',
+    isAuthenticated,
+    async (req, res, next) => {
+        try {
+            const results = await projectService.createProjectAccess(
+                req.user!,
+                req.params.projectUuid,
+                req.body,
+            );
+            res.json({
+                status: 'ok',
+                results,
+            });
+        } catch (e) {
+            next(e);
+        }
+    },
+);
+projectRouter.patch(
+    '/projectAccess',
+    isAuthenticated,
+    async (req, res, next) => {
+        try {
+            const results = await projectService.updateProjectAccess(
+                req.user!,
+                req.params.projectUuid,
+                req.body,
+            );
+            res.json({
+                status: 'ok',
+                results,
+            });
+        } catch (e) {
+            next(e);
+        }
+    },
+);
+projectRouter.delete(
+    '/projectAccess/:userUuid',
```

Comment:
extending my above comment:

here it's gone for the `/access/{userUuid}` approach - alternatively it could use an access record id